app/src/main/java/org/oppia/android/app/recyclerview/BindableAdapter.kt

```diff
@@ -206,6 +206,8 @@ class BindableAdapter<T : Any> internal constructor(
         object : BindableViewHolder<T>(binding.root) {
           override fun bind(data: T) {
             setViewModel(binding, data)
+            // Attach lifecycleOwner after viewModel has been attached to the view.
+            // Attaching lifecycleOwner before will cause a null pointer exception crash on [HomeFragment]
```

Comment:
For here & the version below, two things:
1. I don't think we can actually link to HomeFragment from here since this adapter doesn't import it (plus, markdown links aren't meant to work in single-line comments unlike KDocs).
2. Instead of talking about HomeFragment, could these comments explain why the NPE can sometimes happen? E.g. something like: "Attaching the lifecycle owner before view model initialization can sometimes cause a NullPointerException because X."

Reply:
Makes sense Ben have updated comments.